src/components/ExercisePreview.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback } from "react";
+import { ExerciseData, ExerciseQuestionData, StepBase } from "src/types";
+import { IncludeRemoveQuestion } from "./IncludeRemoveQuestion";
+import { Exercise } from "./Exercise";
+
+export interface ExercisePreviewProps {
+    exercise: ExerciseData;
+    selectedQuestions: string[];
+    setSelectedQuestions?: React.Dispatch<React.SetStateAction<string[]>>;
+    enableOverlay?: boolean;
+}
+
+/** 
+ * An Exercise version with less interaction with card and grants an Overlay with Include/Remove component
+ */ 
+export const ExercisePreview = (
```

Comment:
I'd prefer this to take `selected` and `onSelectionChange` props, that keeps it flexible on the consuming side